.madgerc

```diff
@@ -1,8 +1,8 @@
 {
-    "fileExtensions" : ["ts"],
```

Comment:
Why?

Reply:
`.madgerc` was using a mix of 4 spaces and tabs which rendered badly in my editor. Other jsons in the project are also using 2 spaces so I thought it is appropriate.

Reply:
Ah, I saw it wrong, I thought you changed it to 4 spaces.